packages/date-time-utilities/src/dateGrid/getDayGrid.ts

```diff
@@ -54,14 +54,18 @@ export const getDayGrid = (options: IDayGridOptions): IDay[][] => {
   // in work week view if the days aren't contiguous we use week view instead
   const selectedDateRangeType = getDateRangeTypeToUse(dateRangeType, workWeekDays, firstDayOfWeek);
 
-  let selectedDates = getDateRangeArray(
-    selectedDate,
-    selectedDateRangeType,
-    firstDayOfWeek,
-    workWeekDays,
-    daysToSelectInDayView,
-  );
-  selectedDates = getBoundedDateRange(selectedDates, minDate, maxDate);
+  let selectedDates: Date[] = [];
+
+  if (!!selectedDate) {
```

Comment:
Please why do we need that check?

Reply:
If we don't do this check, then the dayGrid will always return at least one selected day. That should not be the case initially though.

Reply:
@lorejoh12 , please would that break anything on v7 Datepicker?

Reply:
I think this should be fine. Normal usage that doesn't pass in an initial selected date wouldn't necessarily be expected to have a selected date range. Also from testing the examples in the deployed test page, the existing examples that do have an initially selected date (week selection, month selection) still initially render fine, so I think this should be okay.

Signing off for this specific change